conda_build/build.py

```diff
@@ -1619,8 +1619,13 @@ def bundle_conda(output, metadata, env, stats, **kw):
     elif files:
         # Files is specified by the output
         # we exclude the list of files that we want to keep, so post-process picks them up as "new"
+        include = [f for f in files if not f.startswith('!')]
+        exclude = [f[1:] for f in files if f.startswith('!')]
```

Comment:
We also need to consider files which themselves start with a `!`. For example with `gitignore` you can start entries with `\!` for this.
Personally, I'd rather suggest to have a more explicit form to avoid these custom syntax and special cases  (also wrt. YAML tags). I.e., I'd rather have `files` optionally be a dictionary with two keys:
```yaml
  files:
    include:
      - *.txt
    exclude:
      - do-not-want.txt
```
.

Reply:
Agreed, @mbargull!